### PR TITLE
Add run for ImageNet pre-training with fewer classes

### PR DESCRIPTION
**Pull request by avital**
This lets us train only on the subset of ImageNet classes that don't overlap
with CIFAR10 classes.

### CODE REVIEW

**Comment by avital** [comment]
(I am starting the training run now)

**Comment by craffel** [comment]
The class list matches what I have locally, without horse-cart as intended. Commands look good, minor nit that it may be a little clearer to not set the `--consistency_model` flag because we are not using SSL here (it will be set to `mean_teacher` by default, but its value doesn't actually matter since `max_consistency_coefficient=0`.

**Comment by craffel** [comment]
(will approve once we confirm it trains and something reasonable happens)

**Comment by avital** [comment]
Training accuracy ended up at 56.3%, compared to 54.6% for the normal unfiltered run. (I didn't run eval on the non-filtered run recently so I can't compare those). This is expected -- we have fewer training examples so with the same regularization one would expect slightly higher training accuracy.

I'm starting the fine-tuning run now.

**Comment by avital** [comment]
We have results on "filtered ImageNet without CIFAR classes -> CIFAR" finetuning.
It gets 12.91% test error on best validation error

Compared to 12.0% as we report in the paper on "Full ImageNet -> CIFAR" fine-tuning
It's still better than all of the SSL techniques on the same model.

So we should probably report both and explain. And we can consider adding the confusion matrix experiment, or not. I am open to both.
